fvtest/compilertriltest/MaxMinTest.cpp

```diff
@@ -159,3 +185,138 @@ INSTANTIATE_TEST_CASE_P(MaxMin, Int64MaxMin, ::testing::Combine(
     ::testing::Values(
         std::make_tuple<const char*, int64_t(*)(int64_t, int64_t)>("lmax", lmax),
         std::make_tuple<const char*, int64_t(*)(int64_t, int64_t)>("lmin", lmin))));
+
+class FloatMaxMin : public TRTest::BinaryOpTest<float> {};
+
+TEST_P(FloatMaxMin, UsingConst) {
+    std::string arch = omrsysinfo_get_CPU_architecture();
+    SKIP_IF(OMRPORT_ARCH_X86 == arch || OMRPORT_ARCH_S390 == arch || OMRPORT_ARCH_S390X == arch || OMRPORT_ARCH_HAMMER == arch || OMRPORT_ARCH_AARCH64 == arch, KnownBug)
+        << "The " << arch << " code generator currently doesn't support fmax/fmin (see issue #4176)";
```

Comment:
In other tests, the issue number in `SKIP_IF` messages refers to an issue documenting the why the test to be skipped. It allows someone who sees the message to get more information if they so desire. Since #4176 doesn't actually deal with the lack of support for fmax/fmin, maybe a new issue should be opened and the messages (this one and the others like it in this change) should reference that one instead?

Reply:
Sure, I can create a new Issue and reference it.

Reply:
https://github.com/eclipse/omr/issues/4276